pkg/grub/Dockerfile

```diff
@@ -18,32 +18,33 @@ RUN apk add --no-cache \
   autoconf \
   pkgconf \
   patch \
-  gettext-dev
-RUN [ "$(uname -m)" != riscv64 ] || apk add coreutils
+  gettext-dev \
+  coreutils
 
 # because python is not available
 RUN ln -s python3 /usr/bin/python
 
 # list of grub modules that are portable between x86_64 and aarch64
 ENV GRUB_MODULES_PORT="part_gpt fat ext2 iso9660 squash4 gzio linux acpi normal cpio crypto disk boot crc64 gpt \
-search_disk_uuid search_part_label search_label verify xzio xfs video gfxterm serial gptprio chain probe reboot regexp smbios    \
+search_disk_uuid search_part_label search_label xzio xfs video gfxterm serial gptprio chain probe reboot regexp smbios    \
 part_msdos cat echo test configfile loopback"
 ENV GRUB_MODULES_i386_pc="multiboot multiboot2 biosdisk"
-ENV GRUB_MODULES_x86_64="multiboot multiboot2 efi_uga efi_gop linuxefi"
+ENV GRUB_MODULES_x86_64="multiboot multiboot2 efi_uga efi_gop"
 ENV GRUB_MODULES_aarch64="xen_boot efi_gop"
-ENV GRUB_MODULES_riscv64="part_gpt fat ext2 iso9660 squash4 gzio linux acpi normal cpio crypto disk boot crc64 \
-search_label xzio xfs video gfxterm serial  chain probe reboot regexp smbios part_msdos cat echo test configfile loopback"
-ENV GRUB_COMMIT=71f9e4ac44142af52c3fc1860436cf9e432bf764
+ENV GRUB_MODULES_riscv64="part_gpt fat ext2 iso9660 squash4 gzio linux acpi normal cpio crypto disk boot crc64 gptprio search_disk_uuid \
+search_part_label search_label xzio xfs video gfxterm serial chain probe reboot regexp smbios part_msdos cat echo test configfile loopback"
+# grub-2.06 tag
+ENV GRUB_COMMIT=ae94b97be2b81b625d6af6654d3ed79078b50ff6
```

Comment:
It seems you're changing the GRUB to 2.06 for all architectures -- you should only do it for riscv64 (and for arm64 if you want to work with @Insei on testing). x86 MUST stay as-is for now (until we prove that the 2.06 GRUB is stable for us)

Reply:
Okay, will make it so, such that the grub commit is only set for RISCV64